include/_pokey.h

```diff
@@ -7,6 +7,7 @@
 /*                                                                           */
 /*                                                                           */
 /* (C) 2000 Freddy Offenga <taf_offenga@yahoo.com>                           */
+/* 06-Nov-2018: Christian Krueger: Added defines for keyboard codes          */
```

Comment:
As far as I know/see none of the cc65 C header files contains a comment like this so far. So I'm wondering you are willing to contribute without it. In general I personally don't see the point in having "inband" change history given the omnipresence of version control systems.

Reply:
There is "prior art" like that in _antic.h. But I'd prefer to have it formatted (and _antic.h changed accordingly) like in e.g. apple2.h.

Reply:
Thanks for the hint :-) I fully agree: The 'apple2.h' style is fine - if considered necessary in the first place.

Reply:
I've removed the info but thought about if removing all personal credits doesn't make more sense?
(A wrong information is more worse than none.)
Wouldn't make something like "(c) The cc65 team and contributors (https://www.cc65.org/doc/CREDITS)" make more sense? (Of course with a link to an updated list here.)

Reply:
I personally rather dislike both those credits in the header files as well as the credit/history section at the very top of many/most asm source files for exactly the reason given: They are wrong/outdated. I'm personally open to removing any of those. **HOWEVER**,  I can imagine that those things are important for some contributor or another so I suggest to not remove anything as they just don't bother me/us enough to risk any irritation of even a single contributor. But at least I personally never add anything anywhere, I just don't care.

Reply:
As the main maintainer and so being rather "visible" you can talk... ;)
But what about reestablishing the central credits and linking to them? A contributor could add himself to the credits if he feels so.
I have zero problem with not being mentioned, but giving the copyright in the header to a person which isn't fully responsible for the content doesn't feels right.
Any opinions from others?

Reply:
Trying to bring that topic forward...

What about this: Someone (@IrgendwerA8 ?) checks that credits file and adds those who are named in any header file. Then the header files are changed as proposed. I'm willing to merge such a PR without broad consultation.

Reply:
Good proposal. Being quite thoroughly, maybe @polluks could chime in?